packages/sommelier.rb

```diff
@@ -214,17 +273,14 @@ def self.build
               --xwayland-path=/usr/local/bin/Xwayland \
               --xwayland-gl-driver-path=#{CREW_LIB_PREFIX}/dri \
               --peer-cmd-prefix=#{CREW_PREFIX}#{@peer_cmd_prefix} \
-              --enable-xshape \
-              --noop-driver \
+              --noop-driver ${xdg_shell_v6} \
               --no-exit-with-child \
-              /bin/sh -c \\"touch ~/.Xauthority
-                xauth -f ~/.Xauthority add ${DISPLAY} . $(xxd -l 16 -p /dev/urandom)
-                source #{CREW_PREFIX}/etc/sommelierrc\\""
-            echo $sommelier_cmd >> #{CREW_PREFIX}/var/log/sommelier.log
-            $sommelier_cmd &>> #{CREW_PREFIX}/var/log/sommelier.log
+              /bin/sh -c ${xauth_cmd}"
+            echo "$sommelier_cmd" >> #{CREW_PREFIX}/var/log/sommelier.log
```

Comment:
Only echo with `-v`?

Reply:
This is going to the log file, so you can tell what the command line used to start sommelier was. It's incredibly useful for debugging, and doesn't show up on the console.

Reply:
Fair enough but it would be nice to see on the console too, if desired, with a `-v` or `-d` switch option.